quality_modifiers.rb

```diff
@@ -125,7 +125,7 @@ def modifiers
       }),
 
       Modifier.new("Empty README", "The README is the front page of a library. To have this applied you may have a very empty README.", -8, Proc.new { |spec, cd_stats, stats, cp_stats, owners|
-        cd_stats[:readme_complexity].to_i < 25 || spec.documentation_url != nil
+        cd_stats[:readme_complexity].to_i < 25 && spec.documentation_url != nil
```

Comment:
Ah no, the logic was wrong, if you have a bad readme and no external documentation you get the hit.